LINUX.md

```diff
@@ -301,10 +315,10 @@ Open your terminal and go to your `dotfiles` project:
 
 ```bash
 cd ~/code/<YOUR_GITHUB_NICKNAME>/dotfiles
-stt # Open it in Sublime Text
+vsc # Open it in VScode
```

Comment:
I think that in the data program on kitt, the text editor command is still `code .`. Introducing an alias would require to update the program and check in the content there is no more `code .`. IMHO, it doesn't worth it for students. Teachers can do as they please of course.

Reply:
```suggestion
code . # Open it in VS Code
```